### ISSUE TITLE
Show a background blur on the HUD

### ISSUE THREAD

**Issue by YousufSSyed** [AlexPerathoner/SlimHUD#150]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

no

### Describe the solution you'd like

iOS' volume slider has a background blur for the translucent part, SlimHUD should add an option on each HUD to show a background blur as well.
![proxy-image](https://github.com/user-attachments/assets/b2cd84a8-f2c3-46d0-ad3c-a189c375877b)

**Comment by AlexPerathoner** [comment]
Might the "Shadow" setting in the Design tab be what you're looking for?
![immagine](https://github.com/user-attachments/assets/f9529c1b-d5bd-4ec6-80fc-533d3c24c472)

**Comment by YousufSSyed** [comment]
No. Notice in the picture I put, the image behind that iOS volume slider is blurred.

**Comment by AlexPerathoner** [comment]
Oh, now I see what you mean. I'll look into it

**Comment by AlexPerathoner** [comment]
Just released in v1.5.3 #162
Thanks Zorth!

**Comment by zorth64** [comment]
> Just released in v1.5.3 [#162](https://github.com/AlexPerathoner/SlimHUD/pull/162) Thanks Zorth!

You're welcome, @AlexPerathoner. I'm very happy to be able to help improve SlimHUD and make it even better. 🫡
